inspirehep/submissions/loaders.py

```diff
@@ -7,10 +7,40 @@
 
 """Submissions Loaders"""
 
-from invenio_records_rest.loaders.marshmallow import marshmallow_loader
+from flask import request
+from invenio_records_rest.loaders.marshmallow import (
+    MarshmallowErrors,
+    marshmallow_loader,
+)
+
+from inspirehep.submissions.errors import RESTDataError
 
 from .marshmallow import Author, Job, Literature
 
+
+def inspire_marshmallow_loader(schema_class):
```

Comment:
for all the other cases we are using marshmallow_loader, why we need a custom one?

Reply:
We are not using marshmallow loaders, please check. It's not compatible with our submission views.